spec/channels/room_channel_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe RoomChannel, type: :channel do
+  before do
+    stub_connection user_id: user.id
```

Comment:
You can put this before in just one line:
```
before { stub_connection(user_id: user.id) }
```

Also, I think the `let` statements should go above the before.